### PR TITLE
Feature/hyp upgrade

### PR DESCRIPTION
**Pull request by jvinolas**
Upgrade alpine in hypervisor from 3.8 to 3.9 that will upgrade libvirt from 4.4 to 4.10 and qemu from 2.2 to 3.1

### CODE REVIEW

File: dockers/hypervisor/Dockerfile [hunk 1]
```diff
@@ -2,9 +2,17 @@ FROM isard/alpine-pandas:1.0.0
...
 
 RUN pip3 uninstall pandas pytz python-dateutil six -y
+RUN sed -i -e 's/v3\.8/v3.9/g' /etc/apk/repositories
+RUN apk update
+RUN apk upgrade --available
 
 RUN apk --no-cache add qemu-system-x86_64 libvirt netcat-openbsd libvirt-daemon dbus polkit qemu-img
-RUN ln -s /usr/bin/qemu-system-x86_64 /usr/bin/qemu-kvm
+RUN apk --no-cache add alsa-utils alsa-lib qemu-audio-alsa
+ADD dockers/hypervisor/asoundrc /root/.asoundrc
```
Comment (NefixEstrada) [inline dockers/hypervisor/Dockerfile]: We should try to switch all the `ADD` statements from the Dockerfiles to `COPY`. It's a best practice, you can check it [here](https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#add-or-copy)

File: dockers/hypervisor/Dockerfile [hunk 2]
```diff
@@ -2,9 +2,17 @@ FROM isard/alpine-pandas:1.0.0
 MAINTAINER isard <info@isard.com>
 
 RUN pip3 uninstall pandas pytz python-dateutil six -y
+RUN sed -i -e 's/v3\.8/v3.9/g' /etc/apk/repositories
```
[the remark this answers is not in the document]
Comment (jvinolas) [reply dockers/hypervisor/Dockerfile]: I know we should upgrade the pandas alpine base we are using with the rest of images but I tried and it broke other containers, so a lot of work will be needed to fix that. My thoughts is to work at least with latests hypervisor packages so we can see that the virtualization is working as expected. Meanwhile I'll work in fixing the rest of images.

File: dockers/hypervisor/Dockerfile (hunk 1 above)
Comment (jvinolas) [reply dockers/hypervisor/Dockerfile]: Sure, and as I see threre are audio alsa packages not needed. I'll push again this branch!

**Comment by NefixEstrada** [comment]
Closing in favor of #160

**Pull request closed without merging**